### PR TITLE
Initialisation json view

### PR DESCRIPTION
**Pull request by MJCallahanPage**
**App-Config-Base PR**:
- https://github.com/hmrc/app-config-base/pull/7013

### CODE REVIEW

**Comment by MJCallahanPage** [comment]
https://user-images.githubusercontent.com/16954819/162411291-24289633-675c-47b1-bc49-d88b5838f93a.mov

File: conf/application.conf [hunk 1]
```diff
@@ -65,7 +65,7 @@ microservice {
     upload-customs-documents-frontend {
       protocol = http
       host = localhost
-      port = 10110
+      port = 10100
```
Comment (frobian21) [inline conf/application.conf:68]: Did you want to include this change to use the real UDF locally?

File: conf/application.conf [hunk 2]
```diff
@@ -120,4 +120,6 @@ play.i18n.langs = ["en"]
 #  gtm.container = "<INSERT YOUR GTM CONTAINER LETTER HERE>"
 #}
 
-
+host = "http://localhost:10111"
+upload-customs-documents-frontend.url = "http://localhost:10100"
```
Comment (frobian21) [inline conf/application.conf:124]: same as above

File: conf/application.conf (hunk 1 above)
Comment (MJCallahanPage) [reply conf/application.conf:68]: Ah, yeah, I did this for some local proving.

What might be good, is to have both configs in there, with a feature toggle. So we can switch between calling the existing service and the new one at will.

For now, I think this one should go back to 10110. So I'll update that :)

File: conf/application.conf (hunk 2 above)
Comment (MJCallahanPage) [reply conf/application.conf:124]: as above also 🚀

**Review by frobian21 — APPROVED** [review]

**Pull request merged**